cmd/node-joiner/testdata/add-nodes -report.txt

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
The file name seems to have an extra space `cmd/node-joiner/testdata/add-nodes -report.txt` i.e. add-nodes` `-report.txt